nx-X11/config/cf/X11.tmpl

```diff
@@ -2428,6 +2424,7 @@ ProjectUnsharedLibReferences(WINDOWSWM,NX_WindowsWM,$(WINDOWSWMLIBSRC),XBuildLib
 #ifndef ProfileLibXau
 #define ProfileLibXau		NO	/* profiled auth library */
 #endif
+      XDMCPLIBSRC = $(LIBSRC)/Xdmcp
```

Comment:
I guess your editor did something funny here?

This won't work, as the PR explicitly removes $(LIBSRC)/Xdmcp.